src/main/kotlin/lotto/view/InputView.kt

```diff
@@ -0,0 +1,33 @@
+package lotto.view
+
+import lotto.Lotto
+import lotto.LottoMachine
+
+
+object InputView {
+
+    fun inputPurchaseAmount(): LottoMachine {
+        println("구입 금액을 입력해 주세요.")
+        val price = readln().toIntOrNull() ?: throw IllegalArgumentException("가격 입력이 올바르지 않습니다.")
+        return LottoMachine(price).also { lotto ->
+            println("로또 ${lotto.count}개를 구매했습니다.")
+            lotto.lottos.forEach { lotto ->
```

Comment:
a객체에서 b프로퍼티를 꺼내서 직접 연산(forEach)를 도는것은 객체의 캡슐화를 깨트릴 수 있어요. 
코틀린에서 val 과 immutable List를 활용해 불변성을 제거했지만, 이 경우에도 다음과 같이 내부 프로퍼티를 직접 꺼내지 않고 동작하도록 해보면 어떨까요?

```suggestion
            lotto.forEachNumbers{ lotto -> ...}
```